### PR TITLE
Add new procedures `dbms.cluster.recreateDatabase`, `dbms.cluster.statusCheck`

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1816.surge.sh

Updated pages:
- [Procedures](https://neo4j-docs-operations-1816.surge.sh/operations-manual/5/reference/procedures)

File: modules/ROOT/pages/reference/procedures.adoc [hunk 1]
```diff
@@ -318,6 +318,16 @@ In 4.2, signature changed to `db.listLocks() :: (mode :: STRING, resourceType ::
 | label:yes[]
 | label:new[Introduced in 5.23] label:server-management[`SERVER MANAGEMENT` privilege only]
 
+| xref:reference/procedures.adoc#procedure_dbms_cluster_recreateDatabase[`dbms.cluster.recreateDatabase()`]
+| label:no[]
+| label:yes[]
+| label:new[Introduced in 5.24]
```
Comment (AnnaSjerling) [inline modules/ROOT/pages/reference/procedures.adoc:324]: Would it make sense to include the required privileges here as well? You need to have the same privileges as for dropping and creating a database.

File: modules/ROOT/pages/reference/procedures.adoc [hunk 2]
```diff
@@ -1979,6 +1989,33 @@ m|WRITE
...
+| Description
+a|Recreates a database while keeping all RBAC settings.
+The procedure initiates a process, which when complete, will have synchronized and started all database instances within the cluster.
+| Signature
+m|dbms.cluster.recreateDatabase(database :: STRING, options = {} :: MAP)
+| Mode
+a|WRITE
+|===
+
+[procedure_dbms_cluster_statusCheck]]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/reference/procedures.adoc:2006]:
```suggestion
[[procedure_dbms_cluster_statusCheck]]
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1816.surge.sh

Updated pages:
- [Procedures](https://neo4j-docs-operations-1816.surge.sh/operations-manual/5/reference/procedures)

File: modules/ROOT/pages/reference/procedures.adoc (hunk 1 above)
Comment (NataliaIvakina) [reply modules/ROOT/pages/reference/procedures.adoc:324]: Usually we add the 'Admin only' label if you need `Admin` privileges.

Comment (AnnaSjerling) [reply modules/ROOT/pages/reference/procedures.adoc:324]: Just above this procedure, there are a bunch of procedures with "label:server-management[`SERVER MANAGEMENT` privilege only]". I thought that might be in the same realm as DROP and CREATE database privileges?

Comment (NataliaIvakina) [reply modules/ROOT/pages/reference/procedures.adoc:324]: yep, I know that we have several procedures with this label `SERVER MANAGEMENT`. But in fact this was an exception.
So, you mean this type of privileges - https://neo4j.com/docs/operations-manual/current/authentication-authorization/dbms-administration/#access-control-dbms-administration-database-management, don't you?

Comment (NataliaIvakina) [inline modules/ROOT/pages/reference/procedures.adoc:324]:
```suggestion
| label:new[Introduced in 5.24] label:database-management[`DATABASE MANAGEMENT` privilege only]
```

Comment (NataliaIvakina) [reply modules/ROOT/pages/reference/procedures.adoc:324]: or we can talk about the privileges on another page, where we'll explain how to use this procedure

Comment (AnnaSjerling) [reply modules/ROOT/pages/reference/procedures.adoc:324]: Yes 😄  Is that not done?

Comment (NataliaIvakina) [reply modules/ROOT/pages/reference/procedures.adoc:324]: sorry, what do you mean? 😁

Comment (AnnaSjerling) [reply modules/ROOT/pages/reference/procedures.adoc:324]: So in the docs [here](https://neo4j.com/docs/operations-manual/current/authentication-authorization/dbms-administration/#access-control-dbms-administration-database-management), at Table 23, I think we need the first two columns. But do you mean this is something which is usually written in text in the documenatation part instead?

Comment (AnnaSjerling) [reply modules/ROOT/pages/reference/procedures.adoc:324]: In a discussion elsewhere: The privileges will only be in the recreate doc under "Managing databases in a cluster" and "Recreate databases"

**Review by AnnaSjerling — APPROVED** [review]
Nice!

**Pull request merged**